diff --git a/node_modules/react-app-rewire-multiple-entry/lib/utils.js b/node_modules/react-app-rewire-multiple-entry/lib/utils.js
index 352e99f..285b842 100644
--- a/node_modules/react-app-rewire-multiple-entry/lib/utils.js
+++ b/node_modules/react-app-rewire-multiple-entry/lib/utils.js
@@ -19,7 +19,7 @@ var formatName = function formatName(name) {
     name = '';
   }

-  return name.split('/').reverse()[0].match(/^[^.]*/)[0] + '.' + H.update(name).digest().toString(16);
+  return name.split('/').reverse()[0].match(/^[^.]*/)[0];
 };

 exports.formatName = formatName;
diff --git a/node_modules/react-app-rewire-multiple-entry/src/utils.ts b/node_modules/react-app-rewire-multiple-entry/src/utils.ts
index 36bd9eb..7977744 100644
--- a/node_modules/react-app-rewire-multiple-entry/src/utils.ts
+++ b/node_modules/react-app-rewire-multiple-entry/src/utils.ts
@@ -11,11 +11,7 @@ export const formatName = function (name: string) {
     name
       .split('/')
       .reverse()[0]
-      .match(/^[^.]*/)[0] +
-    '.' +
-    H.update(name)
-      .digest()
-      .toString(16)
+      .match(/^[^.]*/)[0]
   );
 };

